Adding package-lock to 1.7 guide (#4513)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/core-upgrade/00-upgrading-to-v1.7.md

- Don't add a `freshness:` block.
- Explicitly set `freshness: null`

> From v1.7 on, when `dbt deps` is run, it will create/update the `package-lock.yml` file in the _project_root_ where `packages.yml` is recorded. The `package-lock.yml` file contains a record of all packages installed and, if subsequent `dbt deps` runs contain no updated packages in `depenedencies.yml` or `packages.yml`, dbt-core installs from `package-lock.yml`. 

Author: ```suggestion
From v1.7 on, when [`dbt deps`](/reference/commands/deps) is run, it will create or update the `package-lock.yml` file in the _project_root_ where `packages.yml` is recorded. The `package-lock.yml` file contains a record of all packages installed and, if subsequent `dbt deps` runs contain no updated packages in `depenedencies.yml` or `packages.yml`, dbt-core installs from `package-lock.yml`. 
```

> From v1.7 on, when [`dbt deps`](/reference/commands/deps) is run, it will create or update the `package-lock.yml` file in the _project_root_ where `packages.yml` is recorded. The `package-lock.yml` file contains a record of all packages installed and, if subsequent `dbt deps` runs contain no updated packages in `depenedencies.yml` or `packages.yml`, dbt-core installs from `package-lock.yml`. 

Reviewer: ```suggestion
Beginning with v1.7, running [`dbt deps`](/reference/commands/deps) creates or updates the `package-lock.yml` file in the _project_root_ where `packages.yml` is recorded. The `package-lock.yml` file contains a record of all packages installed and, if subsequent `dbt deps` runs contain no updated packages in `depenedencies.yml` or `packages.yml`, dbt-core installs from `package-lock.yml`. 
```
